src/libraries/Microsoft.Extensions.Http/src/Logging/LoggingScopeHttpMessageHandler.cs

```diff
@@ -10,13 +10,20 @@
 
 namespace Microsoft.Extensions.Http.Logging
 {
+    /// <summary>
+    /// Handles logging of the lifecycle for a http request within a scope.
```

Comment:
Let's specify what the "scope" is, so it won't be confused with DI scope.
```suggestion
    /// Handles logging of the lifecycle for an HTTP request within a log scope.
```